UI/test_change_first_name.py

```diff
@@ -1,8 +1,8 @@
-from pages.base_page import BasePage
-from pages.login_page import LoginPage
-from pages.profile_page import ProfilePage
-from pages.edit_profile_page import EditProfilePage
-from configs import link, email, password, new_first_name
+from .pages.base_page import BasePage
+from .pages.login_page import LoginPage
+from .pages.profile_page import ProfilePage
+from .pages.edit_profile_page import EditProfilePage
+from .configs import link, email, password, new_first_name
```

Comment:
I managed to run the UI tests, but this import is not resolved. I guess you didn't add it, because you used real data from the production server.

Ideally, we should start both BE and FE locally and run tests using this sandboxed instances w/o real data.

These are just thoughts to keep in mind, approving

Reply:
Oh, sorry, I missed out your latest commit